### ISSUE TITLE
Does not work in Windows. It restarts Apache when ray function is called.

### ISSUE THREAD

**Issue by atiqsamtia** [CuyZ/global-ray#3]
I used this path for prepend in Windows.

```
auto_prepend_file=C:\Users/Atiq/AppData/Roaming/Composer/vendor/cuyz/global-ray/prepend_ray.php
```

the path is correct and any PHP file works file except the one which has any `ray()` function call.

**Comment by romm** [comment]
Hi @atiqsamtia,

Unfortunately I don't have any Windows environnement

**Comment by romm** [comment]
Hi @atiqsamtia,

Unfortunately I don't have any Windows environment to test this package out at the moment. I'm not sure this would be an issue but did you try to replace slashes `/` with backslashes `\` in the configuration path?

Could you also dump a `phpinfo()` in your script and see if the `auto_prepend_file` configuration is filled correctly?

**Comment by atiqsamtia** [comment]
Hi @romm

Sorry I'm off to bed. I'll share more details in the morning.

The issue occurs only when Ray function is called. It just restarts the apache in a loop. Weired behavior.

**Comment by atiqsamtia** [comment]
Tested with both `/` and `\\`

confirmed in phpinfo() as well prepend file is there.

![image](https://user-images.githubusercontent.com/5112753/113998764-cf818e00-9872-11eb-9c5b-fafba5e12c4a.png)

I'm not sure why is it happening. It keeps restarting my XAMPP and I can see my ports getting free and then assigned again.

**Comment by romm** [comment]
Strange issue indeed. Could you try to add a `die` statement juste before `require_once` here to check if the file is even loaded?

https://github.com/CuyZ/Global-Ray/blob/be3e1172ee7565a2fa1409c5f32328125f2b85ed/prepend_ray.php#L6

**Comment by atiqsamtia** [comment]
Yes it does load the file and I can see the output of `die('hi');`

**Comment by romm** [comment]
Hi @atiqsamtia, I honestly have no idea what could possibly go wrong. I also don't have access to a Windows machine to try it out by my side at the moment, so it will be hard for me to help you solving this issue. 

Sorry for the inconvenience, if you find anything related to this issue, please consider keeping us updated here.

Have a good day. 😉
